src/app/features/new-petition/new-petition-issue/new-petition-issue.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, EventEmitter, OnInit, Output } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { shareReplay, tap } from 'rxjs';
+import { NewPetitionIssueService } from 'src/app/logic/petition/exports';
+import { IssuePetitionData, Result } from 'src/app/shared/models/exports';
+
+@Component({
+  selector: 'dp-new-petition-issue',
+  templateUrl: './new-petition-issue.component.html',
+})
+export class NewPetitionIssueComponent implements OnInit {
+  public formGroup: FormGroup;
+  protected result$;
+  protected loading$;
+
+  @Output() _cancelEvent: EventEmitter<void> = new EventEmitter();
```

Comment:
@dcrosa1993 let's avoid using starting underscore for the component's public API.